aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -123,6 +146,94 @@ function makePut {
 	fi
 }
 
+function getGatewayid {
+    GATEWAY=$(makeGET servers body| jq "[.[].id]")
+}
+
+function createLabel_if_does_not_exist {
+    CREATE_LABEL=$1
+    ENCODED_LABEL=${CREATE_LABEL// /%20}
+    JQ_LABEL_FILTER='.[] | select([.name=="'$CREATE_LABEL'"] | any) | .name'
+    RESPONSE_LABEL=$(makeGet settings/labels body  | jq "$JQ_LABEL_FILTER")
+    if [ -n "$RESPONSE_LABEL" ]; then
+        log "Label \"$CREATE_LABEL\" already exists."
+    else
+        log "Label \"$CREATE_LABEL\" does not exist, creating it."
+        NEW_LABEL=$(makePost settings/labels "{ \"name\": \"$CREATE_LABEL\" }" | jq ".name")
+        if [ "$NEW_LABEL"="$CREATE_LABEL" ]; then
```

Comment:
If it does not match it creates in previous if statement here: https://github.com/anshupande/dcos-scripts/blob/c4d0d93fa1b9e935815e23381cc857d7f870f22e/aqua-ethos-setup/aqua_ethos.sh#L164